web/packages/dmt-core/src/utils/truncatePathString.ts

```diff
@@ -17,10 +17,10 @@ export function truncatePathString(
   let counter: number = 0
   while (truncatedPath.length <= maxLength) {
     previousPath = truncatedPath
-    truncatedPath = truncatedPath.replace(
-      '...',
-      `${splitTextOnFolders[counter]}/...`
-    )
+    //truncatedPath = truncatedPath.replace(
+    //  '...',
+    //  `${splitTextOnFolders[counter]}/...`
+    // )
```

Comment:
is there a bug here? maybe create new issue for it

Reply:
IDK :man_shrugging:  Came from the rebase :incoming_envelope: